scripts/generate.sh

```diff
@@ -75,6 +84,10 @@ protoc_with_imports gogoslick encoding/x/protobuf/internal/wirepb/wire.proto
 protoc_with_imports gogoslick internal/examples/protobuf/examplepb/example.proto
 protoc_with_imports yarpc-go internal/examples/protobuf/examplepb/example.proto
 
+ragel -Z -G2 -o internal/interpolate/parse.go internal/interpolate/parse.rl
```

Comment:
It doesn't. I just tried to use process substitution to do this instead but that mangles the file names in the `//line` directives in the generated code like so,

```
//line /dev/fd/63:9
const interpolate_start int = 8
const interpolate_first_final int = 8
const interpolate_error int = 0
```